console/module/tag/src/main/java/org/eclipse/kapua/app/console/module/tag/client/TagFilterPanel.java

```diff
@@ -37,7 +39,7 @@ public TagFilterPanel(AbstractEntityView<GwtTag> entityView, GwtSession currentS
         super(entityView, currentSession);
         entityGrid = entityView.getEntityGrid(entityView, currentSession);
         this.currentSession = currentSession;
-
+        setIcon(new KapuaIcon(IconSet.FILTER));
```

Comment:
We probably want to move this line in the base class constructor, so all the filters inherits it. Check also if other specialized filters sets it and remove it from there.